src/adic_space.lean

```diff
@@ -380,29 +475,122 @@ to_stalk.rec (presheaf_of_rings.restrict U G) u (stalk_of_rings G u)
     refl,
   end))
 
-instance {X : Type*} [topological_space X] {U : opens X} (G : presheaf_of_rings X) (u : U) :
+instance {X : Type u} [topological_space X] {U : opens X} (G : presheaf_of_rings X) (u : U) :
   is_ring_hom (presheaf_of_rings.restrict_stalk_map G u) :=
 by unfold presheaf_of_rings.restrict_stalk_map; apply_instance
 
-def presheaf_of_topological_rings.restrict {X : Type*} [topological_space X] (U : opens X)
+def presheaf_of_topological_rings.restrict {X : Type u} [topological_space X] (U : opens X)
   (G : presheaf_of_topological_rings X) : presheaf_of_topological_rings U :=
   { Ftop := λ V, G.Ftop (topological_space.opens.map U V),
     Ftop_ring := λ V, G.Ftop_ring (topological_space.opens.map U V),
     res_continuous := λ V W HWV, G.res_continuous (topological_space.opens.map U V)
       (topological_space.opens.map U W) (topological_space.opens.map_mono HWV),
   ..presheaf_of_rings.restrict U G.to_presheaf_of_rings }
 
-noncomputable def 𝒞.restrict {X : Type*} [topological_space X] (U : opens X) (G : 𝒞 X) : 𝒞 U :=
+noncomputable def 𝒞.restrict {X : Type u} [topological_space X] (U : opens X) (G : 𝒞 X) : 𝒞 U :=
 { F := presheaf_of_topological_rings.restrict U G.F,
   valuation :=
     λ u, Spv.mk (valuation.comap (G.valuation u).out (presheaf_of_rings.restrict_stalk_map _ _)) }
 
-structure adic_space (X : Type u) [topological_space X] :=
+noncomputable instance PreValuedRingedSpace.restrict {X : PreValuedRingedSpace.{u}} :
+  has_coe (opens X) PreValuedRingedSpace :=
+{ coe := λ U,
+  { space := U,
+    top := by apply_instance,
+    presheaf := presheaf_of_topological_rings.restrict U X.presheaf,
+    valuation :=
+      λ u, Spv.mk (valuation.comap (X.valuation u).out (presheaf_of_rings.restrict_stalk_map _ _)) } }
+
+section 𝒱
+local attribute [instance] sheaf_of_topological_rings.uniform_space
+
+/-- Wedhorn's category 𝒱 -/
```

Comment:
Now I'm confused. I thought Wedhorn's 𝒱 was somewhere above?

Reply:
Wedhorn's `𝒱` is the same as our `CLVRS`. I've added this in the docstring of `CLVRS`. If you and @kbuzzard agree we can remove all the code about `𝒱`.

Reply:
I think C and V are very "mathsy" names. Clearly computer scientists are not fond of these names. I'm happy either way.

Reply:
I've removed all code about `𝒱`.